README.md

```diff
@@ -12,6 +12,71 @@ transit_model is a rust crate to manage transit data. Its model corresponds to t
 cargo build --release
 ```
 
+### Compile for KV1
+The KV1 format needs a additional dependency called [Proj.4](https://proj4.org/)
+which allows the transformation of localization coordinates.
+[crates.io](https://crates.io/) provides a
+[`proj`](https://crates.io/crates/proj) crate which is a binding to the C
+library (version 6.0.0). This means you need this version installed on your
+system. The commands below will build and install [Proj.4](https://proj4.org/) on your system.
+
+```
+apt install \
+	wget \
+	build-essential \
+	automake autoconf libtool pkg-config \
+	sqlite3 libsqlite3-dev
+cd /tmp
+wget https://github.com/OSGeo/proj.4/releases/download/6.0.0/proj-6.0.0.tar.gz
+tar -xzvf proj-6.0.0.tar.gz
+cd proj-6.0.0
+./configure --prefix=/usr
+make
+make install
+```
+
+[Proj.4](https://proj4.org/) is configured as a `feature` of the `transit_model`
+crate.  Once [Proj.4](https://proj4.org/) is installed, you need a few more
+dependencies for building `transit_model`.
+```
+apt install -y \
```

Comment:
I remember `clang-dev` unblocking one compilation error I had. However, I had a lot and in the end, I'm not 100% sure which one solved which. I'll remove that.

Reply:
I played with a lot of dependencies and `libcland-dev` did solve one of my problem at some point. But maybe for the wrong reasons. I'll remove it since I saw you didn't need it in the `Dockerfile`. I agree about not leaving stuff that is not needed. And if it really is, we'll add it another time when people mention it.